algorithmic_efficiency/logger_utils.py

```diff
@@ -262,7 +262,7 @@ def _get_workload_properties(workload: spec.Workload) -> dict:
   return workload_properties
 
 
-def get_meta_data(workload: spec.Workload) -> dict:
+def get_meta_data(workload: spec.Workload, rng_seed: int = None) -> dict:
```

Comment:
I think using `dict` as a type hint is only supported for Python >= 3.9. So I think we can replace it with `Dict`, and when the types are known, we should also specify them, e.g. `Dict[str, int]`. Same thing for every use of `dict` as a type hint in this file.

Reply:
Also, you can replace `rng_seed: int = None` with `rng_seed: Optional[int] = None`.

Reply:
Thanks you're right. I replaced them with Dict for now.